script/rest/update-files.js

```diff
@@ -105,51 +107,11 @@ async function getBundledFiles(schemas) {
   // name of the `github/github` checkout. A CI test
   // checks the version and fails if it's not a semantic version.
   for (const filename of schemas) {
-    const schema = JSON.parse(await readFile(path.join(DEREFERENCED_DIR, filename)))
+    const schema = JSON.parse(await readFile(path.join(DOCS_DEREF_OPENAPI_DIR, filename)))
 
     schema.info.version = `${githubBranch} !!DEVELOPMENT MODE - DO NOT MERGE!!`
-    await writeFile(path.join(DEREFERENCED_DIR, filename), JSON.stringify(schema, null, 2))
-  }
-}
-
-// Gets the full list of unpublished, deprecated, and active schemas
-// from the github/github repo
-async function getSchemas() {
-  const openAPIConfigs = await readdir(OPEN_API_RELEASES_DIR)
-  const unpublished = []
-  const deprecated = []
-  const currentReleases = []
-
-  // The file content in the `github/github` repo is YAML before it is
-  // bundled into JSON.
-  for (const file of openAPIConfigs) {
-    const newFileName = `${path.basename(file, 'yaml')}deref.json`
-    const content = await readFile(path.join(OPEN_API_RELEASES_DIR, file), 'utf8')
-    const yamlContent = yaml.load(content)
-    if (!yamlContent.published) {
-      unpublished.push(newFileName)
-    }
-    if (yamlContent.deprecated) {
-      deprecated.push(newFileName)
-    }
-    if (!yamlContent.deprecated && yamlContent.published) {
-      currentReleases.push(newFileName)
-    }
-  }
-
-  const allSchemas = { currentReleases, unpublished, deprecated }
-  if (versions) {
-    await validateVersionsOptions(allSchemas)
-    return versions.map((elem) => `${elem}.deref.json`)
-  }
-  const schemas = allSchemas.currentReleases
-  if (includeUnpublished) {
-    schemas.push(...allSchemas.unpublished)
-  }
-  if (includeDeprecated) {
-    schemas.push(...allSchemas.deprecated)
+    await writeFile(path.join(DOCS_DEREF_OPENAPI_DIR, filename), JSON.stringify(schema, null, 2))
   }
-  return schemas
```

Comment:
}

 async function getBundlerOptions() {
  let includeParams = []
  if (versions) {
    includeParams = versions
  }
  if (includeUnpublished) {
    includeParams.push('--include_unpublished')
  }
  if (includeDeprecated) {
    includeParams.push('--include_deprecated')
  }
  return includeParams.join(' ')
}
async function validateInputParameters() {
  // The `--versions` and `--decorate-only` options cannot be used
  // with the `--include-deprecated` or `--include-unpublished` options
  const numberOfOptions = Object.keys(program.opts()).length
  if (numberOfOptions > 1 && (decorateOnly || versions)) {
    const errorMsg = `🛑 You cannot use the versions and decorate-only options with any other options.\nThe decorate-only switch will decorate all dereferenced schemas files in the docs-internal repo.\nThis script doesn't support generating individual deprecated or unpublished schemas.\nPlease reach out to #docs-engineering if this is a use case that you need.`
    throw new Error(errorMsg)
  }
  // Check that the github/github repo exists. If the files are only being
  // decorated, the github/github repo isn't needed.
  if (!decorateOnly) {
    try {
      await stat(GITHUB_REP_DIR)
    } catch (error) {
      const errorMsg = `🛑 The ${GITHUB_REP_DIR} does not exist. Make sure you have a codespace with a checkout of \`github/github\` at the same level as your \`github/docs-internal \`repo before running this script. See this documentation for details: https://thehub.github.com/epd/engineering/products-and-services/public-apis/rest/openapi/openapi-in-the-docs/#previewing-changes-in-the-docs.`
      throw new Error(errorMsg)
     }
   }
 }